setup.py

```diff
@@ -16,6 +16,16 @@
 
 import kalite
 
+# Since pip 7.0.1, bdist_wheel has started to be called automatically when
```

Comment:
It's backwards compatible... any pip should work. Pip is not a big part of installations, older versions would just fetch a tar.gz and install it with setuptools.... but somehow the 7 version wants to build a .whl first with the `wheel` package.